app/src/main/res/layout/profile_activity.xml

```diff
@@ -1,44 +1,214 @@
+
+                <TextView
+                    android:id="@+id/org_name"
+                    android:layout_width="153dp"
+                    android:layout_height="28dp"
+                    android:layout_marginTop="56dp"
+                    android:layout_marginEnd="92dp"
+                    android:layout_marginRight="92dp"
+                    android:layout_marginBottom="8dp"
+                    android:text="TextView"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintVertical_bias="0.0" />
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/forConsumer"
+                android:layout_width="match_parent"
+                android:layout_height="174dp">
+
+                <ImageView
+                    android:id="@+id/blurred_consumer_pic"
+                    android:layout_width="0dp"
+                    android:layout_height="174dp"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginLeft="4dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginRight="8dp"
+                    android:layout_marginBottom="8dp"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="1.0"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintVertical_bias="0.003"
+                    app:srcCompat="@android:drawable/screen_background_light_transparent" />
+
+                <ImageView
+                    android:id="@+id/iv_prof_pic"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginLeft="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginRight="8dp"
+                    app:layout_constraintBottom_toBottomOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="0.492"
+                    app:layout_constraintStart_toStartOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintTop_toTopOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintVertical_bias="0.252"
+                    app:srcCompat="@drawable/instagram_user_outline_24" />
+
+                <TextView
+                    android:id="@+id/tv_name"
+                    android:layout_width="wrap_content"
```

Comment:
these heights and margins are kind of all over the place, i think you'll want to clean them up and make them consistent between the consumer/org layouts (probably)